src/tests/ftest/dfuse/pil4dfs_fio.yaml

```diff
@@ -0,0 +1,75 @@
+hosts:
+  test_servers: 2
+  test_clients: 1
+
+timeout: 600
+
+server_config:
+  name: daos_server
+  engines_per_host: 2
+  engines:
+    0:
+      pinned_numa_node: 0
+      fabric_iface: ib0
+      fabric_iface_port: 31317
+      log_file: daos_server0.log
+      storage:
+        0:
+          class: dcpm
+          scm_list: ["/dev/pmem0"]
+          scm_mount: /mnt/daos0
+    1:
+      pinned_numa_node: 1
+      fabric_iface: ib1
+      fabric_iface_port: 31417
+      log_file: daos_server1.log
+      storage:
+        0:
+          class: dcpm
+          scm_list: ["/dev/pmem1"]
+          scm_mount: /mnt/daos1
+
+pool:
+  size: 90%
+
+container:
+  type: POSIX
+  control_method: daos
+
+test_pil4dfs_vs_dfs:
+  bw_deltas:
+    read: 10
+    write: 10
+
+fio:
+  names:
+    - global
+    - job
+  output_format: 'json'
+  global:
+    direct: 1
+    iodepth: 1
+    time_based: 1
+    runtime: '60s'
+    ramp_time: '5s'
+    group_reporting: 1
+    cpus_allowed_policy: split
+    numa_mem_policy: bind:all
+  job:
+    params: !mux
+      small_fork:
```

Comment:
does small/large here refer to the block size?
it would be better in this case to use something clearer for the job name, so maybe call it 256b_fork, 1m_fork, etc.
That way it becomes easier in case we want to add more IO size variations in the future.

Reply:
Yes small and large was refering to the block size.

- [ ] Update mux name of test size

Reply:
fixed with commit 4cee92787